packages/block-library/src/template-part/edit/inner-blocks.js

```diff
@@ -46,5 +48,12 @@ export default function TemplatePartInnerBlocks( {
 			alignments: themeSupportsLayout ? alignments : undefined,
 		},
 	} );
-	return <TagName { ...innerBlocksProps } />;
+
+	return (
```

Comment:
I think the changes here could potentially break alignments inside template parts.

Reply:
Ive mostly been testing on a header including a nested template part, and the alignments seem to be the same as before and correspond to the front-end output.  I definitely wouldn't want to break any alignments though, is there anything specific use case that might make this problematic?

Reply:
I think this new approach should be a bit more safe here.